src/component/Popup.js

```diff
@@ -8,13 +8,43 @@ import CancelIcon from "@mui/icons-material/Cancel"; // Importing the Cancel ico
 import Select from '@mui/material/Select';
 import MenuItem from '@mui/material/MenuItem';
 
+import { toast } from 'react-toastify';
+const REACT_APP_SERVER_URL = "http://localhost:8181";
+
 const Popup = ({ open, handleClose }) => {
   const tags = [
     { id: 1, name: "Angular", color: "error" },
     { id: 1, name: "Vue", color: "success" },
     { id: 1, name: "React", color: "primary" },
   ];
 
+  const handleSave = async (e) => {
+    const title = document.getElementById('outlined-basic').value;
```

Comment:
Please use some specific class name for eg:- snippet-title, snippet-code, snippet-tag or sl-title